nnpdf_data/nnpdf_data/filter_utils/legacy_jets_utils.py

```diff
@@ -0,0 +1,1146 @@
+
+
+def process_err_ATLAS_2JET_7TEV_R06(error, cv):
+    """
+    Converts an error given in percentage
+    of the central data value in absolute value.
+
+    Note: the d'Agostini prescription for the
+    symmetrization of the error does not hold here.
+    We follow here the experimental prescription
+
+    Parameters
+    ----------
+
+    error : dictionary
+            e.g. {'label': 'sys', 'symerror': 0.1%}
+
+    cv : float
+        central value
+
+    Returns
+    -------
+    tuple
+        tuple containing two floats
+
+    """
+    if "lum" in error:
+        # luminosity uncertainty is always symmetric
+        sigma = float(error['symerror'].strip('%')) / 100.0 * cv
+        return sigma
+
+    elif error['label'] == 'sys':
+        if 'asymerror' in error:
+            d_p = float(error['asymerror']['plus'].strip('%')) / 100.0 * cv
+            d_m = float(error['asymerror']['minus'].strip('%')) / 100.0 * cv
+
+            tmp1 = d_p
+            tmp2 = d_m
+            # case 1: d_p and d_m are both negative
+            if tmp1 < 0.0 and tmp2 < 0.0:
+                if tmp2 < tmp1:
+                    d_p = 0.0
+                    d_m = tmp2
+                else:
+                    d_p = 0.0
+                    d_m = tmp1
+
+            # case 2: d_p and d_m are both positive
+            if tmp1 > 0.0 and tmp2 > 0.0:
+                if tmp1 > tmp2:
+                    d_p = tmp1
+                    d_m = 0.0
+                else:
+                    d_p = tmp2
+                    d_m = 0.0
+            return d_p / np.sqrt(2.0), d_m / np.sqrt(2.0)
+
+        else:
+            sigma = float(error['symerror'].strip('%')) / 100.0 * cv
+            return sigma / np.sqrt(2.0), -sigma / np.sqrt(2.0)
+
+
+def HEP_table_to_df_ATLAS_2JET_7TEV_R06(heptable, scenario='nominal'):
+    """
+    Given hep data table return a pandas
+    DataFrame with index given by Ndata,
+    columns by the uncertainties and
+    np.nan entries
+
+    Parameters
+    ----------
+    heptable : str
+            path to hepdata table
+
+    scenario : 0, 1, 2
+            0 = nominal, 1 = stronger, 2 = weaker
+    """
+
+    with open(heptable) as file:
+        card = yaml.safe_load(file)
+
+    # list of len ndata. Each entry is dict with
+    # keys errors and value
+    card = card['dependent_variables'][SCENARIO_ATLAS_2JET_7TEV_R06[scenario]]['values']
+    df = pd.DataFrame(index=range(1, len(card) + 1))
+
+    errors = card[0]['errors']
+    for i, err in enumerate(errors):
+        # luminosity uncertainty, always symmetric
+        if (
+            (scenario == 'nominal' and i == 67)
+            or (scenario == 'stronger' and i == 57)
+            or (scenario == 'weaker' and i == 69)
+        ):
+            df["lum"] = np.nan
+
+        elif err['label'] == 'sys':
+            df[f"{err['label']}_plus_{i}"] = np.nan
+            df[f"{err['label']}_minus_{i}"] = np.nan
+
```

Comment:
Could you do the same here as I did in [2e367b3](https://github.com/NNPDF/nnpdf/pull/2056/commits/2e367b39668ddf3384fca9f00470630b8533f4e2) for the 1jet case?